test/components/DayPicker_spec.jsx

```diff
@@ -871,6 +872,23 @@ describe('DayPicker', () => {
     });
   });
 
+  describe('#getWeekHeaders', () => {
+    it('returns unmutated weekday headers for currentMonth in a future', () => {
+      sinon.stub(PureDayPicker.prototype, 'render');
```

Comment:
```suggestion
      sinon.stub(PureDayPicker.prototype, 'render', () => null);
```
I think the issue is that this stub is causing it to render `undefined`, which isn't valid.